kivy/_event.pyx

```diff
@@ -228,7 +228,19 @@ cdef class EventDispatcher(ObjectWithUid):
         prop_args = {
             k: kwargs.pop(k) for k in list(kwargs.keys()) if k in properties}
         self._kwargs_applied_init = set(prop_args.keys()) if prop_args else set()
-        super(EventDispatcher, self).__init__(**kwargs)
+
+        # at this point any kwargs passed may go to object
+        # which will raise a TypeError and cause confusion
+        try:
+           super(EventDispatcher, self).__init__(**kwargs)
+        except TypeError as e:
+           if kwargs and str(e).startswith("object.__init__() takes exactly one argument"):
```

Comment:
```suggestion
           if kwargs and str(e).startswith("object.__init__() takes"):
```

The CI shows the following failed test:

```
>       assert str(cm.value).startswith(
            "object.__init__() takes exactly one argument")
E       AssertionError: assert False
E        +  where False = <built-in method startswith of str object at 0x17CE8660>('object.__init__() takes exactly one argument')
E        +    where <built-in method startswith of str object at 0x17CE8660> = 'object.__init__() takes no parameters'.startswith
E        +      where 'object.__init__() takes no parameters' = str(TypeError('object.__init__() takes no parameters',))
E        +        where TypeError('object.__init__() takes no parameters',) = <ExceptionInfo TypeError('object.__init__() takes no parameters',) tblen=2>.value

```

Because the message after `takes` can be different.

Reply:
Already fixed it :)